mcp/src/graph/neo4j.ts

```diff
@@ -26,7 +26,7 @@ class Db {
   private driver: Driver;
 
   constructor() {
-    const uri = `neo4j://${process.env.NEO4J_HOST || "localhost"}`;
+    const uri = `neo4j://${process.env.NEO4J_HOST || "localhost"}:7687`;
```

Comment:
dont hardcode like this... on production we set NEO4J_HOST = `neo4.sphinx:7687`. So this would break 

You can add the port to the default if you want

const uri = `neo4j://${process.env.NEO4J_HOST || "localhost:7687"}`;